### PR TITLE
TW 258 Refresh /getting-started/creating-your-first-workspace/

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Adds `search_keyword` to frontmatter 
- Converts narrative into step-by-step instructions
- Checked screenshots - no updates needed

### CODE REVIEW

File: src/pages/docs/getting-started/creating-your-first-workspace.md [hunk 1]
```diff
@@ -14,28 +16,30 @@ contextual_links:
...
 
-<img alt="Change workspace or create new" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.1.jpg" width="300px"/>
+    <img alt="Change workspace or create new" src="https://assets.postman.com/postman-docs/workspace-switcher-v9.1.jpg" width="300px"/>
 
-Click __Create Workspace__ to add a new workspace.
+1. Enter a **Name** and optional **Summary** for your workspace.
+1. Choose the workspace's visibility:
+    * A **Personal** workspace is only visible to you.
+    * A **Private** workspace is only visible to you and to any team members you invite to join it.
+    * A **Team** workspace allows you to invite collaborators.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/creating-your-first-workspace.md:33]:
```suggestion
    * A **Team** workspace is visible to everyone on your team.
```

**Review by mark-anthony-postman — APPROVED** [review]
Made one suggestion--since Private workspaces also allow you to invite collaborators, seems like the description for Team workspaces could be tweaked a bit. Otherwise LGTM--much better page structure than before! 🙂

File: src/pages/docs/getting-started/creating-your-first-workspace.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/creating-your-first-workspace.md]: Good call! Updated.

File: src/pages/docs/getting-started/creating-your-first-workspace.md [hunk 2]
```diff
@@ -14,28 +16,30 @@ contextual_links:
     url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"
 ---
 
-Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share APIs, collections, environments, and other components of your work in Postman.
+Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](https://learning.postman.com/docs/sending-requests/managing-environments/), and other components of your work in Postman.
```
Comment (jkonrath-postman) [inline src/pages/docs/getting-started/creating-your-first-workspace.md:19]:
```suggestion
Postman workspaces allow you to organize and collaborate on API projects with your team. Within each workspace you can share [APIs](/docs/designing-and-developing-your-api/creating-an-api/), [collections](/docs/getting-started/creating-the-first-collection/), [environments](/docs/sending-requests/managing-environments/), and other components of your work in Postman.
```

**Review by jkonrath-postman — APPROVED** [review]
One little fix, but otherwise LGTM!

File: src/pages/docs/getting-started/creating-your-first-workspace.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/creating-your-first-workspace.md:19]: Eep, thanks for catching this! 🙈

**Pull request merged**